base/applications/calc/utl_ieee.c

```diff
@@ -27,7 +27,7 @@ void prepare_rpn_result_2(calc_number_t *rpn, TCHAR *buffer, int size, int base)
 
     switch (base) {
     case IDC_RADIO_HEX:
-        _stprintf(buffer, _T("%I64X"), rpn->i);
+        StringCbPrintf(buffer, sizeof(buffer), _T("%I64X"), rpn->i);
```

Comment:
Wrong: buffer is a pointer, so sizeof(buffer) == 4 and the code just fails. The buffer size is probably given by that `size` parameter (please verify). And check whether it's a size in bytes or in number of characters.

Reply:
OK. The bug is fixed.